supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemAsyncJobExecutionActor.scala

```diff
@@ -220,14 +220,19 @@ trait SharedFileSystemAsyncJobExecutionActor
   }
 
   private def metadataKey(key: String) = MetadataKey(workflowId, Option(metadataJobKey), key)
-
```

Comment:
It can keep putting it back in then. It's muscle memory and I'm not using an auto-formatter which is incorrect ;)